beacon-chain/state/v2/state_trie.go

```diff
@@ -25,14 +25,14 @@ import (
 )
 
 // InitializeFromProto the beacon state from a protobuf representation.
-func InitializeFromProto(st *ethpb.BeaconStateAltair) (*BeaconState, error) {
```

Comment:
Shouldn't we be returning the base BeaconState interface ? I am surprised no tests broke here